Managers/CardManager.gd

```diff
@@ -11,6 +11,8 @@ var card_container: CardContainer = null
 var current_deck: Array[CardBase] = []
 var default_deck: CardSetBase = null
 
+var disable_card_animations: bool = false
```

Comment:
I see this is used as a check later in handle_effect_queue, and in tests.
Should this be moved to a global file ? I think we can keep it here, but it just felt a bit strange

Reply:
Well CardManager is global in a way. I think it kind of fits because it's card related

Reply:
Yeah it's a singleton, but it was more to have all our variables used globally in a same place
Choosing to keep them in their related files also works